DFC.TestAutomation.UI/DFC.TestAutomation.UI.csproj

```diff
@@ -1,14 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Interesting. What does this do?

Reply:
it will force us to address any warnings in the build. will fail instead of warn.  Happy to remove, but I think it seem like good practice